### PR TITLE
NDRCN 295- Updated Models & Tests

### PR DESCRIPTION
**Pull request by simms11**

### CODE REVIEW

File: app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala [hunk 1]
```diff
@@ -61,16 +61,46 @@ object EISCreateCaseRequest {
...
         DutyTypeTaxDetails = request.Content.DutyTypeTaxDetails,
         DocumentList = request.Content.DocumentList
       )
     }
+
+    def getImporterDetails( request: CreateClaimRequest) : EISUserDetails = {
+      val name = UserName(request.Content.ImporterDetails.Name.firstName,
+          request.Content.ImporterDetails.Name.lastName)
+
+      val fullName:String = (name.firstName+ name.lastName).mkString("")
```
Comment (lmugnai) [inline app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala]:
```suggestion
      val fullName:String = (name.firstName + name.lastName).mkString(" ")
```

File: app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala [hunk 2]
```diff
@@ -61,16 +61,46 @@ object EISCreateCaseRequest {
...
+        Address = request.Content.ImporterDetails.Address
+      )
+    }
+
+
+    def getAgentUserDetails(request: CreateClaimRequest): EISUserDetails =  {
+      val name = UserName(request.Content.AgentDetails.get.Name.firstName,
+          request.Content.ImporterDetails.Name.lastName)
+
+      val fullName:String = (name.firstName+ name.lastName).mkString("")
```
Comment (lmugnai) [inline app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala]:
```suggestion
      val fullName:String = (name.firstName + name.lastName).mkString(" ")
```

**Review by lmugnai — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala (hunk 2 above)
Comment (simms11) [reply app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala]: image
@lmugnai  implementing **mkString(" ")** will result within the attached screenshot

Comment (lmugnai) [reply app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISCreateCaseRequest.scala]: OK, thanks for checking. Still a bit unsure, "The mkString() method is utilized to display all the elements of the list in a string along with a separator". How is the space between the 2 being inserted? Also, is firstName (and lastName) a List?

**Pull request merged**